### PR TITLE
Minor design update to marketing task

### LINKED ISSUE #7642: Minor design update to marketing task
### Is your feature request related to a problem? Please describe.
The grey section dividers are too heavy and distract from the content of the marketing task screen. This is a design update to clean up the layout of this screen - The grey category headers are removed and some padding between elements is updated.

image

### PR DESCRIPTION
**Pull request by jacob-sewell**
Remove background color from marketing tool list header, increase padding in plugin list, rearrange list borders.

Fixes #7642 

#### Before
8D426DAC-A05C-4172-9414-F8D42DAAA56D

#### After
6ED30762-3811-40B3-8F57-A5B6D73A489E

### Detailed test instructions:

Open Marketing task, verify styles look like After instead of Before. If changes aren't seen, verify that the independent WooCommerce Admin plugin is being used (see path in System Status report).

### CODE REVIEW

**Comment by elizaan36** [comment]
hey @jacob-sewell 👋
Looking good! I'm seeing slightly different text weights in the body copy on your machine, but looks like the same before and after so I'm not too worried about that.

I'll paste the design here as a reference:
Screen Shot 2021-09-29 at 10 15 28 PM

A couple comments:
- We need less of a gap at the top and bottom of section headings - what are you currently using now?
- In terms of the text style, can we use the `section-heading` style to force all caps and a lighter weight?

**Review by elizaan36 — CHANGES_REQUESTED** [review]
We need slightly less of a gap at the bottom of the section headers. Maybe try:
padding-top: $gap-larger;
padding-bottom: $gap;

This is so it's more clear that the section header is associated with the extensions listed below.

**Comment by elizaan36** [comment]
hey @jacob-sewell 👋
Looking good, just left a comment above on the padding.

One other thing - can we use the `section-heading` text style for the section headings to force caps and lighter weight?

I'll paste the design as a reference.

image

**Comment by jacob-sewell** [comment]
@elizaan36 let me know if this addresses your comments.

**Comment by mattsherman** [comment]
> One other thing - can we use the `section-heading` text style for the section headings to force caps and lighter weight?

We cannot. Gutenberg's Text component doesn't properly handle the styles for variants (I need to open a separate issue about this in the Gutenberg repo).

File: client/tasks/fills/Marketing/Marketing.scss [hunk 1]
```diff
@@ -17,4 +17,19 @@
...
+
+		h3 {
+			text-transform: uppercase;
+		}
+	}
+
+	.woocommerce-plugin-list__plugin {
+		border-top: none;
+		border-bottom: 1px solid #e0e0e0;
+	}
```
Comment (mattsherman) [inline client/tasks/fills/Marketing/Marketing.scss:34]: I believe the correct place for these css changes is actually in `PluginList.scss`.

File: client/task-list/tasks/Marketing/Marketing.scss [hunk 2]
```diff
@@ -17,4 +17,19 @@
...
+
+		h3 {
+			text-transform: uppercase;
+		}
+	}
+
+	.woocommerce-plugin-list__plugin {
+		border-top: none;
+		border-bottom: 1px solid #e0e0e0;
+	}
```
Comment (mattsherman) [inline client/task-list/tasks/Marketing/Marketing.scss:34]: I believe the correct place for these css changes is actually in `PluginList.scss`.

**Review by mattsherman — DISMISSED** [review]
I tested and the changes look good to me. Left comments regarding where the changes should be made.

File: client/task-list/tasks/Marketing/Marketing.scss (hunk 2 above)
Comment (jacob-sewell) [reply client/task-list/tasks/Marketing/Marketing.scss:34]: > I believe the correct place for these css changes is actually in `PluginList.scss`.

But we don't want to apply these changes to other tasks, do we? Here it's nested under `.woocommerce-task-marketing`.

**Review by elizaan36 — APPROVED** [review]
I think it looks good!

**Review by joshuatf — APPROVED** [review]
Looks good; thanks for updating this design, @jacob-sewell!  🚢

File: client/task-list/tasks/Marketing/Marketing.scss (hunk 2 above)
Comment (jacob-sewell) [reply client/task-list/tasks/Marketing/Marketing.scss]: Disregard my objection. I was confused about the class structure (which, to be fair, is confusing) in these files.

**Pull request merged**